### DISCUSSION TITLE
Display wiki page name in tab instead of <%-- [PAGE_INFO] Untitled-...

### DISCUSSION THREAD

**Discussion by JeremiPlazas** [Frederisk/Wikitext-VSCode-Extension#60] [Ideas]
Hi there,

I'm still getting familiar with your extension which i am finding extremely helpful so far! One hiccup when it comes to using it for extensive editing of 100s of different pages and templates on many different wikis is that i often find myself editing many pages at the same time. To have the wiki page names on the tabs seems obvious to me, though i don't know how difficult it may be programatically. Right now all tabs say something like `<%-- [PAGE_INFO] Untitled-...` which is quite useless. Am i maybe missing something? Did i miss a setting somewhere? Or is that not implement?

Thanks so much for your work!
<img width="982" alt="Screen Shot 2022-05-03 at 1 03 14 PM" src="https://user-images.githubusercontent.com/4063437/166524341-995c7d08-c1c6-4252-88c9-3acff5e3bb7c.png">

**Comment by Frederisk** [comment]
This seems to be duplicate of #56?

**Reply by JeremiPlazas** [reply]
Is it? I'm not sure. #56 seems to refer to file names as those pages are saved locally, which i don't really care about. But maybe that's the same thing as the name displayed on the tab. If so yes that would be the same.

**Reply by Frederisk** [reply]
For `TextDocument`, the display title will indeed be the name of the saved file aka [`readonly fileName: string`](https://code.visualstudio.com/api/references/vscode-api#TextDocument). And there is still no progress on [this issue](https://github.com/microsoft/vscode/issues/41909). Please let me know if I missed something.🤔

**Reply by JeremiPlazas** [reply]
That makes sense. Thank you for indulging my ignorant questions! We can close / delete this discussion then, I'll follow #56 for further developments.
